cl/citations/match_citations.py

```diff
@@ -42,11 +42,15 @@ def make_name_param(defendant, plaintiff=None):
     return " ".join(query_words), len(query_words)
 
 
-def reverse_match(conn, results, citing_doc):
+def reverse_match(
+    conn: ExtraSolrInterface, results: List, citing_doc: Opinion
```

Comment:
Might be good to say what this is a `List` of, but in CourtListener I'm only looking for progress not perfection wrt types.

Reply:
Yes, I'll specify that.